qa-automation.md

```diff
@@ -0,0 +1,82 @@
+# Automation QA  - Challenges
+
+## Challenge 1 - Automate 'User Registration' of an website with Selenium
+
+### Test Case 1 - Automate User Registration Process
+
+#### Steps to Automate
+
+1. Open this url: <http://testautomationpractice.blogspot.com/>
+2. Enter your information in Volunteer Sign Up section.
+3. Click on Submit button.
+4. Validate that user is created or failed to create.
+
+### Test Case 2 - Verify invalid email address error
+
+#### Steps to Automate
+
+1. Open this url:  <http://testautomationpractice.blogspot.com/>
+2. Enter invalid email address in the email box and click enter.
+3. Validate that an error message is displaying
+
+### Test Case 3 – Verify Drag and Drop ability on website
+
+#### Steps to Automate
+
+1. Open this url:  <http://testautomationpractice.blogspot.com/>
+2. Write a script to demonstrate Drag and Drop
+
+## Challenge 2 - Automate 'Buy Product' functionality of an e-commerce website with Selenium
+
+### Test Case 1 - Automate End to End Buy Order functionality
+
+#### Steps to Automate
+
+1. Open link <http://practice.automationtesting.in/>
+2. Select a product
+3. Go to Cart
+4. Proceed Check Out
+5. Enter Delivery Information
+6. Proceed Order
+7. Verify Result with Screen Shot.
+
+### Test Case 2 - Verify that Total Price is reflecting correctly if user changes quantity on 'Shopping Cart Summary' Page
+
+#### Steps to Automate
+
+1. Open link <http://practice.automationtesting.in/>
+2. Select Multiple Products
+3. Go to Cart
+4. Change Product quantity.
+5. Verify that Total price is changing and reflecting correct price.
+
+## Challenge 3 - Automate 'Search Product' functionality of an e-commerce website
+
+### Test Case 1 - Automate 'Search Product' Functionality of an e-commerce website with Selenium
+
+#### Steps to Automate
+
+1. Open link <http://practice.automationtesting.in/>
+2. Move your cursor to Search Box
+3. Search “Selenium”
+4. Validate the result – “Selenium Ruby”
+5. Continue to search for “Python”
+6. Validate the result – “Sorry, nothing found.”
+
+## Notes
+
+* When your project is done, please share with us as a public GitHub / Bitbucket.
```

Comment:
@LeoAnhNguyen Can you add "The repository must avoid containing the words prophet, betprophet and challenge."